worker/jobTypes/S3Publish.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs-extra');
+const workerUtils = require('../utils/utils');
+
+class S3PublishClass {
+
```

Comment:
Do we want two objects in the same core that have exactly the same definition? S3 might work more statically -- we pass the GitHubJob into the function.